x-pack/plugins/security/public/views/login/components/basic_login_form.tsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiButton, EuiCallOut, EuiFieldText, EuiFormRow, EuiPanel, EuiSpacer } from '@elastic/eui';
+import React, { ChangeEvent, Component, Fragment } from 'react';
+import { LoginState } from '../../../../common/login_state';
+
+interface Props {
+  http: any;
+  window: any;
+  infoMessage?: string;
+  loginState: LoginState;
+  next: string;
+}
+
+interface State {
+  hasError: boolean;
+  isLoading: boolean;
+  username: string;
+  password: string;
+  message: string;
+}
+
+export class BasicLoginForm extends Component<Props, State> {
+  public state = {
+    hasError: false,
+    isLoading: false,
+    username: '',
+    password: '',
+    message: '',
+  };
+
+  public componentDidMount() {
+    document.addEventListener('keydown', this.submitOnEnter);
+  }
+
+  public componentWillUnmount() {
+    document.removeEventListener('keydown', this.submitOnEnter);
+  }
+
+  public render() {
+    return (
+      <Fragment>
+        {this.renderMessage()}
+        <EuiPanel>
+          <EuiFormRow label="Username">
+            <EuiFieldText
+              name="username"
+              title="username"
```

Comment:
Is there a specific use-case for having a `title` attribute here? If not I would recommend getting rid of this, since it's usually not fulfilling much purpose. Same for the password field.

Reply:
no specific user-case for `title`, I'll remove!